packages/@react-spectrum/checkbox/stories/CheckboxGroup.stories.tsx

```diff
@@ -135,3 +152,32 @@ function ControlledCheckboxGroup() {
     </CheckboxGroup>
   );
 }
+
+function renderWithDescriptionErrorMessageAndValidation() {
+  function Example() {
+    let [checked, setChecked] = useState<string[]>(['dogs', 'dragons']);
+    let isValid = checked.length === 2 && checked.includes('dogs') && checked.includes('dragons');
+  
+    return (
+      <Flex width="480px">
+        <CheckboxGroup
+          label="Pets"
+          onChange={setChecked}
+          value={checked}
+          validationState={isValid ? 'valid' : 'invalid'}
+          description="Select only dogs and dragons."
```

Comment:
Mind changing this to something different from the one of the error messages? It was a bit confusing, I thought the description wasn't being removed at first.

```suggestion
          description="Select a pet."
```